Sources/Graphiti/Types/Field.swift

```diff
@@ -41,11 +41,15 @@ public class FieldBuilder<Root, Context, Type> {
 
     func addAllFields() throws {
         for property in try properties(Type.self) {
-            let field = GraphQLField(
-                type: try schema.getOutputType(from: property.type, field: property.key)
-            )
+            do {
```

Comment:
Yes, I think it has to be explicit.  I prefer do it this way. But need to put documentation somewhere.